src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionDescriptorBuilder.cs

```diff
@@ -138,10 +138,12 @@ public static IList<ControllerActionDescriptor> Build(ApplicationModel applicati
             ActionModel action)
         {
             var defaultControllerConstraints = Enumerable.Empty<IActionConstraintMetadata>();
+            var defaultControllerEndpointMetadata = Enumerable.Empty<object>();
```

Comment:
`Array.Empty<object>`?

Reply:
`Enumerable.Empty` also returns an empty array. I'll leave it as enumerable for consistency with the line above 🤷‍♂️